src/time_tracking_lmo.jl

```diff
@@ -20,6 +20,38 @@ TimeTrackingLMO(blmo::BoundedLinearMinimizationOracle) =
 TimeTrackingLMO(blmo::BoundedLinearMinimizationOracle, int_vars) =
     TimeTrackingLMO(blmo, Float64[], Int[], Int[], 0, int_vars)
 
+#========================================================================================#
+is_decomposition_invariant_oracle(tlmo::TimeTrackingLMO) = is_decomposition_invariant_oracle(tlmo.blmo)
+
+function compute_inface_extreme_point(tlmo::TimeTrackingLMO, direction, x; lazy, kwargs...)
+    tlmo.ncalls += 1
+    free_model(tlmo.blmo)
+    sblmo = tlmo.blmo.simple_lmo
```

Comment:
The BLMO does not necessarily have a field `simple_lmo`. The `MathOptBLMO`, for example, doesn't.